sample-compose-fragments/build.gradle.kts

```diff
@@ -72,4 +75,12 @@ dependencies {
     androidTestImplementation("androidx.compose.ui:ui-test-junit4:${rootProject.extra["compose_version"]}")
     debugImplementation("androidx.compose.ui:ui-tooling:${rootProject.extra["compose_version"]}")
     debugImplementation("androidx.compose.ui:ui-test-manifest:${rootProject.extra["compose_version"]}")
+    constraints {
+        implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.8.10") {
+            because("kotlin-stdlib-jdk7 is now a part of kotlin-stdlib")
+        }
+        implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.10") {
+            because("kotlin-stdlib-jdk8 is now a part of kotlin-stdlib")
+        }
+    }
 }
```

Comment:
I have checked the Emma's PR these dependencies are in ComposeDependencies.kt and these are in build.gradle.kts; maybe there will be some conflicts.

Reply:
Thanks Ali, yeah I did copy them over from Emma's branch as I also needed to access some of the stuff from the library updates, and needed to add this to get a passing build. I think we can deal with the conflicts when they occur on either my branch or Emma's branch depending on which get's merged down first. :)